CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# 1.0.0 (February 24, 2022)
+
+This is the official release of the Twilio Live Interactive Video iOS and web Apps. This project demonstrates an interactive live video streaming app that uses [Twilio Live](https://www.twilio.com/docs/live) and [Twilio Video](https://www.twilio.com/docs/video).
```

Comment:
Not sure if `official` is correct here. I'm not sure what it means in this context. Maybe `This is the version 1.0.0 release...`? Or `This is the initial release...`?

Also, should we mention that this also uses Sync? It might even be neat to link to @timrozum's blog post too, since it contains a wealth of knowledge about this app (suppose it might even be better in the README): https://www.twilio.com/blog/interactive-live-streaming-app-programmable-video-live-sync

Reply:
Cool, I think that is a good idea. I added the blog post to the README and Twilio Sync links to both the README and CHANGELOG. Please feel free to suggest edits